.gitignore

```
@@ -1 +1,2 @@
-.env
\ No newline at end of file
+.env
+.docker/
```

Do jeito que está aqui, se quisermos adicionar configurações versionadas no diretório `.docker` não poderemos. Vamos ignorar apenas volumes de dados. Inclusive acho que pode
```suggestion
.docker/data/
```

a pasta `.docker` já pode ser criada com um subdiretório de configuração do banco de dados, e também ter um Dockerfile que configure o banco de dados via build e não direto pelo dockerfile. Assim essa pasta fica versionada e o controle do volume é criado automaticamente